azurerm/internal/services/compute/dedicated_host_group_resource.go

```diff
@@ -75,7 +75,7 @@ func resourceArmDedicatedHostGroupCreate(d *schema.ResourceData, meta interface{
 	resourceGroupName := d.Get("resource_group_name").(string)
 
 	if d.IsNewResource() {
-		existing, err := client.Get(ctx, resourceGroupName, name)
+		existing, err := client.Get(ctx, resourceGroupName, name, compute.InstanceView)
```

Comment:
`InstanceView` returns a different response from the API - presumably this wants to be an empty string?

Reply:
Will do